src/sentry/eventstore/base.py

```diff
@@ -196,24 +194,3 @@ def get_prev_event_id(self, event, filter):
         filter (Filter): Filter
         """
         raise NotImplementedError
-
-    def bind_nodes(self, object_list, node_name="data"):
```

Comment:
We never switched to using this function, removing it so the 2 versions don't diverge.